backend/services/history.py

```
@@ -264,6 +264,43 @@ async def delete_generation(
+        versions_mod.delete_versions_for_generation(generation.id, db)
+
+        # Remove the main audio file if it somehow made it to disk.
+        if generation.audio_path:
+            audio_path = config.resolve_storage_path(generation.audio_path)
+            if audio_path is not None and audio_path.exists():
+                try:
+                    audio_path.unlink()
+                except OSError:
+                    # Best-effort cleanup — don't abort the whole sweep
+                    # if a single file can't be removed.
+                    pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

(SIM105)